test/resolver_test.cr

```diff
@@ -3,10 +3,9 @@ require "./test_helper"
 module Shards
   class ResolverTest < Minitest::Test
     def test_find_resolver_with_unordered_dependency_keys
-      dependency = Dependency.new("test", {
-        "branch" => "master",
-        "git"    => "file:///tmp/test",
-      })
+      dependency = Dependency.new("test", branch: "master",
+        resolver: "git", url: "file:///tmp/test",
+      )
```

Comment:
The test doesn't make sense anymore, unless it's changed to parse a YAML file with unordered keys, maybe.

Reply:
It still tests the behaviour of `Shards.find_resolver`.